event-svc/src/event/service.rs

```diff
@@ -145,15 +149,18 @@ impl EventService {
     /// In the future, we will need to do more event validation (verify all EventID pieces, hashes, signatures, etc).
     pub(crate) async fn parse_discovered_event(
         item: &ReconItem<EventId>,
+        source: Option<String>,
```

Comment:
why is source optional?  Aren't all events going to have a source?

Reply:
not when we insert from a migration. 
API: self
Generated time event: self
Recon: the `NodeId.try_from_peer_id(peer_id)`
migration: None